diff --git a/lottie/src/main/java/com/airbnb/lottie/model/layer/ImageLayer.java b/lottie/src/main/java/com/airbnb/lottie/model/layer/ImageLayer.java
index 6cd60b64..aa16e8ca 100644
--- a/lottie/src/main/java/com/airbnb/lottie/model/layer/ImageLayer.java
+++ b/lottie/src/main/java/com/airbnb/lottie/model/layer/ImageLayer.java
@@ -7,6 +7,7 @@ import android.graphics.Matrix;
 import android.graphics.Paint;
 import android.graphics.Rect;
 import android.graphics.RectF;
+import android.util.Log;
 
 import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
@@ -54,7 +55,11 @@ public class ImageLayer extends BaseLayer {
       dst.set(0, 0, (int) (bitmap.getWidth() * density), (int) (bitmap.getHeight() * density));
     }
 
-    canvas.drawBitmap(bitmap, src, dst, paint);
+    try {
+      canvas.drawBitmap(bitmap, src, dst, paint);
+    } catch (RuntimeException e) {
+      Log.e("Sharpen", e.getMessage());
+    }
     canvas.restore();
   }
 
